### DISCUSSION TITLE
Petite Vue Starter

### DISCUSSION THREAD

**Discussion by johannschopplich** [vuejs/petite-vue#21] [Show and tell]
I have created a minimalist starter kit for Petite Vue: https://github.com/johannschopplich/petite-vue-starter

### Key Features
- 🖖 Petite Vue (of course)
- 🍃 Tailwind
- ⚡️ Vite
- 🦋 Typescript
- 🦺 ESLint & Prettier

### Demo

👉 [petite-vue-starter.jhnn.dev](https://petite-vue-starter.jhnn.dev)

**Comment by rguttersohn** [comment]
Thanks for doing this!
